docs/admin/recipes/testing-flex-saml-sso-sptest.md

```
@@ -72,12 +72,13 @@ Refer to the Janssen Server [SAML SSO](https://docs.jans.io/head/janssen-server/
 
 ## FAQ
 
-Check KC scheduler log for any TR issue 
+Check KC scheduler log for any TR issue
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)